src/routes/account/[uid]/+page.server.ts

```diff
@@ -41,6 +47,10 @@ export const actions = {
             }
         }
         console.log("Creating Stripe session");
+
+        const subscriptionData = !allSubscriptions || !hasCustomerSubscribedBefore(allSubscriptions, PRODUCT_CODE) ? {trial_period_days: 30} : {};
```

Comment:
according to the types on the Stripe library `allSubscriptions` will always be truthy, because if there are no subscriptions it returns an empty array. Worth testing, but if that is the case then the `!allSubscriptions` part will never take

Reply:
Wait I just realized this is because _we_ are possibly returning null from `getAllCustomerSubscriptions`. We can fix that by just making it an empty array instead of null

Reply:
done 👍